### PR TITLE
magento/merchdocs#1142: My Wish List | Add the clarification about creating a Wish List from storefront

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

Fix pull request fixes https://github.com/magento/merchdocs/issues/1142

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

-  https://docs.magento.com/user-guide/customers/account-dashboard-my-wish-list.html?itm_source=merchdocs&itm_medium=search_page&itm_campaign=federated_search&itm_term=My%20Wish%20List

whatsnew
Updated [My Wish List](https://docs.magento.com/user-guide/customers/account-dashboard-my-wish-list.html) to include information about how customers can update the wish list from different areas on the storefront.

### CODE REVIEW

File: src/customers/account-dashboard-my-wish-list.md [hunk 1]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
 ![]({% link images/images-ee/account-dashboard-wishlist.png %}){: .zoom}
 _My Wish List_
 
+{:.ee-only}
+## Create a new wish list
+
+In the storefront, Wish List can be created from your account dashboard, product pages, catalog pages and from the shopping cart.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:13]:
```suggestion
In the storefront, a customer can create a wish list  from their account dashboard, a product page, a catalog page, and the shopping cart.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 2]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+### Method 1: From a customer account
+
+1. In the sidebar of your account dashboard, choose **My Wish List**.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:17]:
```suggestion
1. In the sidebar of their account dashboard, the customer chooses **My Wish List**.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 3]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. In the upper-right corner, click **Create New Wish List**.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:19]:
```suggestion
1. In the upper-right corner, clicks **Create New Wish List**.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 4]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. Enter the Wish List Name.
+
+1. If you want others to be able to see your wish list, select the **Public Wish List** checkbox.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:23]:
```suggestion
1. To allow others to see their wish list, selects the **Public Wish List** checkbox.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 5]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. When complete, click <span class="btn">Save</span>.
+
+    ![]({% link images/images-ee/account-dashboard-wishlist-create-new.png %}){: .zoom}
+    _Create New Wish List_{:.ee-only}
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:28]: Whole section is `ee` so not needed here

```suggestion
    _Create New Wish List_
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 6]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+### Method 2: From the catalog page
+
+1. From the storefront, go to the catalog page that contains the product that is to be added to the wish list.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:32]:
```suggestion
1. From the storefront, the customer goes to the catalog page that contains the product they want to add to a wish list.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 7]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. Hover over the product.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:34]:
```suggestion
1. Hovers over the product.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 8]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. Click **Add to Wish List** and select the **Create New Wish List**.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:36]:
```suggestion
1. Clicks the arrow next to the _Add to Wish List_ icon and selects the **Create New Wish List**.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 9]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. Enter the Wish List Name.
+
+1. If you want others to be able to see your wish list, select the **Public Wish List** checkbox.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:40]:
```suggestion
1. To allow others to see their wish list, selects the **Public Wish List** checkbox.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 10]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. When complete, click <span class="btn">Save</span>.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:42]:
```suggestion
1. When complete, clicks <span class="btn">Save</span>.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 11]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+### Method 3: From the Product Detail page
+
+1. From the storefront, go to the detail page of the product that is to be added to the wish list.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:46]:
```suggestion
1. From the storefront, the customer goes to the detail page of the product they want to add to a wish list.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 12]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. Click **Add to Requisition List** and do select the **Create New Wish List**.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:48]:
```suggestion
1. Clicks the arrow next to **Add to Wish List** and chooses **Create New Wish List**.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 13]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. Enter the Wish List Name.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:50]:
```suggestion
1. Enters the **Wish List Name**.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 14]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. If you want others to be able to see your wish list, select the **Public Wish List** checkbox.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:52]:
```suggestion
1. To allow others to see their wish list, selects the **Public Wish List** checkbox.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 15]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. When complete, click <span class="btn">Save</span>.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:54]:
```suggestion
1. When complete, clicks <span class="btn">Save</span>.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 16]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+    ![]({% link images/images-ee/account-dashboard-wishlist-create-from-pdp.png %}){: .zoom}
+    _Create New Wish List from the Product Detail page _{:.ee-only}
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:57]:
```suggestion
    _Create New Wish List from the Product Detail page _
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 17]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+### Method 4: From the shopping cart
+
+1. Open the **Shopping Cart** page.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:61]:
```suggestion
1. Customer opens the **Shopping Cart** page.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 18]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. Under the item, click the **Move to Wishlist**.
+
+1. Select the **Create New Wish List**.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:65]:
```suggestion
1. Under the item, click the arrow next to **Move to Wishlist** and chooses **Create New Wish List**.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 19]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. Enter the Wish List Name.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:67]:
```suggestion
1. Enters the **Wish List Name**.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 20]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. If you want others to be able to see your wish list, select the **Public Wish List** checkbox.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:69]:
```suggestion
1. To allow others to see their wish list, selects the **Public Wish List** checkbox.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 21]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+1. When complete, click <span class="btn">Save</span>.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:71]:
```suggestion
1. When complete, clicks <span class="btn">Save</span>.
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 22]
```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
...
+
+![]({% link images/images-ee/account-dashboard-wishlist-create-from-cart.png %}){: .zoom}
+    _Create New Wish List from the Shopping Cart page _{:.ee-only}
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:74]:
```suggestion
    _Create New Wish List from the Shopping Cart page_
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Let's use third person for customer workflows. Edits for style, grammar, and to reflect Luma theme and demo data.

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling,
Thank you so much for the edits.
I have implemented all suggestions you describe.
Could you please approve?

File: src/customers/account-dashboard-my-wish-list.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-wish-list.md]: Done ✅

File: src/customers/account-dashboard-my-wish-list.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-wish-list.md]: Done ✅

File: src/customers/account-dashboard-my-wish-list.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-wish-list.md]: Done ✅

File: src/customers/account-dashboard-my-wish-list.md (hunk 4 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-wish-list.md]: Done ✅

File: src/customers/account-dashboard-my-wish-list.md (hunk 5 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-wish-list.md]: Done ✅

File: src/customers/account-dashboard-my-wish-list.md (hunk 18 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-wish-list.md]: Done ✅

File: src/customers/account-dashboard-my-wish-list.md (hunk 19 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-wish-list.md]: Done ✅

File: src/customers/account-dashboard-my-wish-list.md (hunk 20 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-wish-list.md]: Done ✅

File: src/customers/account-dashboard-my-wish-list.md (hunk 21 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-wish-list.md]: Done ✅

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1164`.

**Pull request merged**